src/routes/relay/services/relay-limit.service.ts

```diff
@@ -4,25 +4,30 @@ import { ThrottlerStorageService } from '@nestjs/throttler';
 import { ThrottlerStorageRecord } from '@nestjs/throttler/dist/throttler-storage-record.interface';
 
 @Injectable()
-export class RelayLimitService {
-  /** Time to limit in seconds */
+export class RelayLimitService extends ThrottlerStorageService {
```

Comment:
Was it not possible to use it using our first wrapped approach with the ThrottlerStorageService as a private field injected in the constructor?

Reply:
Good catch, this was a leftover from testing issues. I've reverted it as it's definitely better keeping it private.